tasks/overcloud_images.yml

```diff
@@ -0,0 +1,69 @@
+- block:
+    - name: create images destination
+      file:
+        path: /home/stack/images
```

Comment:
Should this path be configurable, or is it always the same for OpenStack?

Reply:
Official docs says to do everything as the "stack" user. This is basically a magic string, though, and at least the "/home/stack" prefix should be factored out into vars/main.yml.